raster/src/main/scala/geotrellis/raster/io/geotiff/reader/GeoTiffInfo.scala

```diff
@@ -160,13 +160,21 @@ object GeoTiffInfo {
             case Tiff =>
               var ifdOffset = byteReader.getInt
               while (ifdOffset > 0) {
-                tiffTagsBuffer += TiffTags.read(byteReader, ifdOffset)(IntTiffTagOffsetSize)
+                val ifdTiffTags = TiffTags.read(byteReader, ifdOffset)(IntTiffTagOffsetSize)
+                // TIFF Reader supports only overviews at this point
+                // Overview is a reduced-resolution IFD
+                val subfileType = ifdTiffTags.nonBasicTags.newSubfileType.flatMap(NewSubfileType.fromCode)
+                if(subfileType.contains(ReducedImage)) tiffTagsBuffer += ifdTiffTags
```

Comment:
The idea is that we're skipping everything that is not a `ReducedImage` (anyway we can't work with anything else)

Reply:
👍  but also noting that this kind of comment is super helpful for a user to see in the method docstring. It would be helpful for the user to know at a glance something like "Only `SubfileType.ReducedImage` tiffs are supported".